src/react-components/link-dialog.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import { FormattedMessage } from "react-intl";
+
+import styles from "../assets/stylesheets/link-dialog.scss";
+
+class LinkDialog extends Component {
+  static propTypes = {
+    linkCode: PropTypes.string
+  };
+
+  render() {
+    if (!this.props.linkCode) {
+      return (
+        <div>
+          <div className={classNames("loading-panel", styles.codeLoadingPanel)}>
+            <div className="loader-wrap">
+              <div className="loader">
+                <div className="loader-center" />
+              </div>
+            </div>
+          </div>
+        </div>
+      );
+    }
+
+    return (
+      <div>
+        <div>
+          <FormattedMessage id="link.in_your_browser" />
+        </div>
+        <div className={styles.domain}>
+          <FormattedMessage id="link.link_domain" />
```

Comment:
Hmm, maybe this should show `https://`, to reinforce that it is actually a URL. 
I would also like it to be an actual `<a>` link, so that I can right click on it and use Firefox's "Send Link to Device" feature or my Pushbullet browser extension, etc.

Reply:
+1 to the <a> that makes sense to me -- I wasn't sure about the `https://` part. I left it out because I don't want users with standalone headsets to think they need to type those characters. (FYI, the redirector is set up so all of these urls work: `http://hub.link`, `https://hub.link`, `http://hubs.link`, and `https:/hubs.link`)